### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by dag13**
This is all of my finished work for week 2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
 
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+All though coronavirus has changed our plans for STEP. This does not mean I can't enjoy my time being apart of the program.
```
Comment (mutisog) [inline README.md:5]: nit: "All though" --> "Although"

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -17,7 +17,9 @@
  */
 function addRandomGreeting() {
   const greetings =
```
Comment (mutisog) [inline portfolio/src/main/webapp/script.js:19]: Consider changing the variable name, now that it doesn't contain greetings.

File: README.md (hunk 1 above)
Comment (mutisog) [inline README.md:5]: nit: "apart" --> "a part"

**Review by mutisog — APPROVED** [review]

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -17,7 +17,11 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
```
Comment (mutisog) [inline portfolio/src/main/webapp/script.js:20]: Consider updating the variable name, now that it doesn't contain any greetings.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -17,7 +17,11 @@
...
+      ['Cheeseburgers are my favorite food', 'I don\'t like peanut butter', 
+       'Blue is my favorite color', 'Heights and cockroaches terrify me',
+       'I was a swimmer in highschool', 'Bojack Horseman is my favorite show on Netflix',
+       'I\'m learning how to kickbox'
+      ];
 
   // Pick a random greeting.
```
Comment (mutisog) [inline portfolio/src/main/webapp/script.js:26]: same: update this comment.

File: README.md (hunk 1 above)
Comment (mutisog) [inline README.md:5]: A few typos: "All though" --> "although", and "apart" --> "a part".

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -8,10 +8,57 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Deron Gentle's Portfolio</h1>
+
+      <img src="/Images/gradpic.jpg" alt ="graduation picture" width="384" height="512">
```
Comment (mutisog) [inline portfolio/src/main/webapp/index.html:13]: Good job on adding alt text! Good practice :)

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -8,10 +8,57 @@
...
+      <h2>Projects</h2>
+
+      <p><u>Text Prediction Algorithm</u>  Date: April 2019
+      <br>Written in C, this algorithm utilizes the trie data structure for efficient text 
+          prediction as well as fast insertion and search of words.
+      </p>
+
+      <p><u>Lonely Party Arrays</u> Date: March 2019
+      <br>An advanced data structure written in Java that automattically allocates and deallocates
+          arrays when needed. This helps to reduces tje unnecessary space that standard arrays take
```
Comment (mutisog) [inline portfolio/src/main/webapp/index.html:31]: typo: "reduce the"

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (dag13) [reply portfolio/src/main/webapp/script.js:20]: Got it I'll update the variable names and comments in a new pull request.

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (dag13) [reply portfolio/src/main/webapp/index.html:13]: Thank you

File: README.md (hunk 1 above)
Comment (dag13) [reply README.md:5]: I just updated the grammatical errors in my cloud shell. When I reach a certain point I will create a new pull request and the changes will be there.

File: portfolio/src/main/webapp/index.html (hunk 6 above)
Comment (dag13) [reply portfolio/src/main/webapp/index.html:31]: Thank you, didn't see that.

**Pull request merged**